scripts/initialize-conductor-data.js

```diff
@@ -115,9 +223,9 @@ async function populateHoloFuelData () {
 populateHoloFuelData()
```

Comment:
Can we have these scripts consistent and not change it depending on the test we need to run locally.